src/handlers/localstorage/nonceManager.ts

```diff
@@ -0,0 +1,13 @@
+import { NonceManager } from '../../entities';
```

Comment:
a pattern we're trying to move to when we update files is importing like: `@rainbow-me/entities`